pkg/provisioner/filestorage/filestorage_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright (c) 2018

Reply:
Thanks, bad copy/pasting on my part haha

Reply:
Do all the files need to be updated?